.github/workflows/integration-dlx.yml

```
@@ -0,0 +1,55 @@
+
+      - name: Install docker-compose
+        run: |
+          python -m pip install --upgrade pip
+          pip install docker-compose==1.29.2
+
+      - name: Start RabbitMQ broker (docker-compose)
+        run: |
+          docker-compose up -d broker
+          # wait for rabbitmq management to be up
+          python - <<'PY'
+import socket, time
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)